rebar.config

```diff
@@ -5,11 +5,22 @@
 
 {extra_src_dirs, ["examples"]}.
 
-{project_plugins, [erlfmt]}.
+{project_plugins, [erlfmt, rebar3_ex_doc]}.
 
-{edoc_opts, [{preprocess, true}]}.
+{ex_doc, [
+    {extras, [
+          {"CHANGELOG.md", #{title => "Changelog"}},
+          {"README.md", #{title => "Overview"}},
+          {"LICENSE", #{title => "License"}}
+    ]},
+    {main, "README.md"},
+    {homepage_url, "https://github.com/grisp/grisp"},
+    {source_url, "https://github.com/grisp/grisp"},
+    {assets, #{"assets" => "assets"}},
+    {api_reference, true}
```

Comment:
This is `true` by default so it doesn't need to be set explicitly.

Reply:
Removed it...